Code/Framework/AzFramework/AzFramework/Asset/AssetCatalog.cpp

```diff
@@ -841,7 +841,7 @@ namespace AzFramework
                     }
 
 #if defined(AZ_ENABLE_TRACING)
-                    if (message.m_assetType == AZ::Data::s_invalidAssetType)
+                    if (isNewAsset && (message.m_assetType == AZ::Data::s_invalidAssetType))
                     {
                         AZ_TracePrintf(
```

Comment:
nit, could possibly convert the AZ_TracePrintf on 846 to AZ_Trace?

Reply:
I'll change it to AZ_Info, but not AZ_Trace. Even though the messages are spammy, they are legitimately notifying of problems. Specifically, we have files in the Cache/ folder that don't have a registered asset type. The worst offenders are the *.abdata.json files that appear to be recent Asset Browser metadata additions, but there are a few others too.